slasher/db/min_max_span.go

```diff
@@ -15,28 +15,28 @@ var highestValidatorIdx uint64
 func saveToDB(validatorIdx uint64, _ uint64, value interface{}, cost int64) {
 	log.Tracef("evicting span map for validator id: %d", validatorIdx)
 
-	err := d.batch(func(tx *bolt.Tx) error {
+	err := d.update(func(tx *bolt.Tx) error {
```

Comment:
tis method might be called from many go routines maybe its better to keep it batch